boards/common/nucleo/Makefile.dep

```diff
@@ -1,3 +1,7 @@
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
+
+ifneq (,$(filter periph_stm32_eth,$(FEATURES_USED)))
+  USEMODULE += luid
```

Comment:
I am certain this is not the right way to do. I need the luid module for mac address generation. Any better ideas?